forms-flow-api/src/formsflow_api/services/form_process_mapper.py

```diff
@@ -230,7 +230,7 @@ def unpublish_previous_mapper(mapper_data: dict) -> None:
         : Should be called with create_mapper method
         """
         try:
-            form_id = mapper_data.get("form_id")
+            form_id = mapper_data.get("previous_form_id") or mapper_data.get("form_id")
```

Comment:
1. need to create a new form version of the current form . so when the new  mapper creates then it will unpublish the previous version. so we need to send previous form id. if previous formid is there will take the data from database against this id.